### PR TITLE
[Updated] - Added rating to notes

### PR DESCRIPTION
**Pull request by BrettKoenig04**

### CODE REVIEW

File: lmn/templates/lmn/notes/note_detail.html [hunk 1]
```diff
@@ -12,4 +12,44 @@ <h2 id="note-page-title">{{ note.show.artist.name }} at {{ note.show.venue.name
       <p id="note-photo"><img src="{{ note.photo.url }}"></p>
 {% endif %}
 
+{% if note.rating == 1 %}
```
Comment (xd3262nd) [inline lmn/templates/lmn/notes/note_detail.html:18]: Should also have the star presented even there is no rating being given yet or write some statement saying no rating

File: lmn/templates/lmn/notes/note_list.html [hunk 2]
```diff
@@ -27,6 +27,47 @@ <h3 class="note-title">{{ note.title }}</h3>
     {% if note.photo %} <!--show if there is a photo-->
       <p id="note-photo"><img src="{{ note.photo.url }}"></p>
     {% endif %}
+
```
Comment (xd3262nd) [inline lmn/templates/lmn/notes/note_list.html:69]: Same over here as well

**Review by xd3262nd — CHANGES_REQUESTED** [review]
Should probably add the user_images directory in your gitignore as well

**Comment by xd3262nd** [comment]
@BrettKoenig04 Not sure if it is a bug but when a user add new note on the app and did not put any rating (or put rating 0) it will not save the note and did not print out any useful message for the user to know they have to put rating on the note.

**Comment by BrettKoenig04** [comment]
> Should probably add the user_images directory in your gitignore as well

Forgot about that, removed items from sub directory. Will do.

File: lmn/templates/lmn/notes/new_note.html [hunk 3]
```diff
@@ -50,6 +50,9 @@ <h2>New note for {{ show.artist.name }} at {{ show.venue.name }} on {{ show.show
           <br>
           5 - Very Satisfied
           </br>
+          <br>
+          {{ form.rating.errors }}
```
Comment (xd3262nd) [inline lmn/templates/lmn/notes/new_note.html]: Works now! Is there a way to highlight this or move this on top of the form so is easily visible?

**Review by xd3262nd — APPROVED** [review]

**Pull request merged**